gns3/main_window.py

```diff
@@ -1105,9 +1105,22 @@ def startupLoading(self):
                 sys.exit(1)
                 return
 
+        run_as_root_path = LocalConfig.instance().runAsRootPath()
+
         if not sys.platform.startswith("win") and os.geteuid() == 0:
+            # touches file to know that user has run GNS3 as root and to prevent
+            # from running as user
+            if not os.path.exists(run_as_root_path):
+                open(run_as_root_path, 'a').close()
```

Comment:
You should catch the OSError exception here in case there is an issue opening the file.